Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -35,7 +35,10 @@
 SAMPLE_SIZE = 2  # number of samples to store in integration context
 EVENTS_INTERVAL_SECS = 60  # interval between events polling
 EVENTS_MODIFIED_SECS = 5  # interval between events status polling in modified
+
 EVENTS_SEARCH_FAILURE_LIMIT = 3  # amount of consecutive failures events search will tolerate
+EVENTS_SEARCH_POLLING_RETRIES = 10  # number of consecutive failures events polling will tolerate
```

Comment:
1. shouldn't this be shorter by default? like, 3 or so? because it can take 10 mins for one retry.
2. the comment is incorrect.

Reply:
1. This is the default value to wait until one search is completed. This could take time, even above 10 minutes.
2. How do you suggest naming it? This is the maximum number of API calls to perform to get the search results. If the search is not completed after it, it will create a new search.